### DISCUSSION TITLE
how to use /workflow

### DISCUSSION THREAD

**Discussion by antongisli** [zed-industries/zed#17205] [Feature Requests]
Hi there, I'm trying to figure out how to use the /workflow feature, getting tired of copy and pasting each code change section from the context window into my code or trying to figure out how to do it inline for each change. There isn't any example in the docs that I could find. Can someone show me some examples perhaps?

Cheers

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
